### PR TITLE
Set up Redux boilerplate and sample actions and reducers

### PR DESCRIPTION
**Pull request by martakule**
Please do not withhold any critique since we need a solid foundation to build the rest of this app.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -36,3 +38,23 @@ class App extends Component {
 }
 
 export default App;
+
+// const mapStateToProps = state => ({
```
Comment (katiedavis) [inline client/src/App.js]: Let's try not to commit commented out code! Keep our files clean 💪

File: client/src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
```
Comment (katiedavis) [inline client/src/actions/index.js]: Why are we using this instead of just fetch? Just curious

**Comment by katiedavis** [comment]
Overall this looks great! I'm curious to see what the others think, and let's make sure to get rid of that commented out code

File: client/src/actions/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import {someUrl} from "../constants/urls"
+
+export const ACTION_TYPES = {
```
Comment (zoeyyandi) [inline client/src/actions/index.js]: Can we put all the action types into a constants file?

**Comment by zoeyyandi** [comment]
Good stuff! I was also wondering why we're using Axios

File: client/src/actions/index.js (hunk 2 above)
Comment (martakule) [reply client/src/actions/index.js]: I don't know. It was already in package.json -- that was my only reason. I'll change to fetch.

File: client/src/actions/index.js (hunk 3 above)
Comment (martakule) [reply client/src/actions/index.js]: Good idea!

File: client/src/App.js (hunk 1 above)
Comment (martakule) [reply client/src/App.js]: Makes sense! I'll fix that.

**Comment by martakule** [comment]
When I export App with this `export default connect(mapStateToProps,mapDispatchToProps)(App);`, I get this error: `Could not find "store" in either the context or props of "Connect(App)". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(App)".`

I don’t know how to fix it, and I won’t comment it out since we don’t do that, so I removed that line and kept `export default App;`

**Comment by martakule** [comment]
The error with "Could not find store" was caused by two `ReactDOM.render()` methods that I failed to notice. Now we have one.

**Pull request merged**